### PR TITLE
Database

### PR DESCRIPTION
**Pull request by Andebugulin**
### UPDATES:

- Added database schema SQL script according to sqema, there:
  -  creating all `TABLES` according to schema
  - inserting `EXAMPLES` to the tables
- updated `postgres.yml` for automating usage of SQL (creating and inserting) script
- automated `./start-services` script for accessing db in the terminal right away, without using localhost:9999

### CODE REVIEW

**Comment by martimolanes** [comment]
> updated postgres.yml for automating usage of SQL (creating and inserting) script

it's an interesting approach, i really like it! Thank you!

> automated ./start-services script for accessing db in the terminal right away, without using localhost:9999

works flawlessly, i really like if the format of the start-service manage_db is like the start-service. Like `./start-service.sh manage-db`

>Added database schema SQL script according to sqema, there:
    creating all TABLES according to schema
    inserting EXAMPLES to the tables

This needs more work to be done, for the moment, this doesn't achieve our specificacion ( you can see it in `docs/database-schema.md` ). The problem is that you are not using the Tags table you specify, for the Task and Activity Table. To change this we need probably to incorporate two new tables where we can store the relation between Tags and Activities/Tasks. **Probably change our specificacion also**. Very good work btw! keep it up :)

File: client/bun.lockb
Comment (martimolanes) [inline client/bun.lockb:1]: dont edit this file in this PR

File: database/init-db.sql [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+-- Create Activity table
+CREATE TABLE IF NOT EXISTS Activity (
+    Id SERIAL PRIMARY KEY,
+    Title VARCHAR(255) NOT NULL,
+    Description TEXT,
+    Url VARCHAR(255),
+    StartDate TIMESTAMP,
+    EndDate TIMESTAMP,
+    Status INTEGER REFERENCES Status(Id),
+    Tags TEXT, -- Consider using an association table for a normalized design
```
Comment (martimolanes) [inline database/init-db.sql]:
```suggestion

```
maybe delete this

File: database/init-db.sql [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+);
+
+-- Create Task table
+CREATE TABLE IF NOT EXISTS Task (
+    Id SERIAL PRIMARY KEY,
+    Name VARCHAR(255) NOT NULL,
+    Content TEXT,
+    StartDate TIMESTAMP,
+    EndDate TIMESTAMP,
+    Tags TEXT, -- Consider using an association table for a normalized design
```
Comment (martimolanes) [inline database/init-db.sql]:
```suggestion
    Tags TEXT, -- Consider using an association table for a normalized design
```
and this

File: database/init-db.sql [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    Status INTEGER REFERENCES Status(Id),
+    ActivityId INTEGER REFERENCES Activity(Id)
+);
+
```
Comment (martimolanes) [inline database/init-db.sql:48]:
```suggestion

CREATE TABLE IF NOT EXISTS ActivityTag (
    ActivityId INTEGER REFERENCES Activity(Id),
    TagId INTEGER REFERENCES Tag(Id),
    PRIMARY KEY (ActivityId, TagId)
);

CREATE TABLE IF NOT EXISTS TaskTag (
    TaskId INTEGER REFERENCES Task(Id),
    TagId INTEGER REFERENCES Tag(Id),
    PRIMARY KEY (TaskId, TagId)
);
```
and add here two tables like this ones

File: docs/database-usage.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+### Accessing the Data
+
+- **Through Adminer**:
+  - Open a web browser and navigate to `http://localhost:9999`.
+  - Log in with the PostgreSQL credentials defined in your `postgres.yml`.
+  - According to your postgres.yml file, the credentials should be:
+
+    -  System: `PostgreSQL`
+    -  Server: `db`
+    - Username: `postgres` (the default superuser account created by the PostgreSQL Docker image)
```
Comment (martimolanes) [inline docs/database-usage.md]: fix format

File: start-service.sh [hunk 5]
```diff
@@ -9,6 +9,7 @@ function help() {
     echo "usage:"
     echo "  run:"
     echo "    $0 db               - start PostgreSQL database"
+    echo "    $0 manage_db        - start a PostgreSQL terminal session"
```
Comment (martimolanes) [inline start-service.sh]: it would be better to have the same format that the script name. `manage-db`

File: server/bun.lockb
Comment (martimolanes) [inline server/bun.lockb:1]: dont edit this file also

**Review by martimolanes — CHANGES_REQUESTED** [review]

File: start-service.sh (hunk 5 above)
Comment (Andebugulin) [reply start-service.sh]: fixed

File: database/init-db.sql (hunk 3 above)
Comment (Andebugulin) [reply database/init-db.sql:48]: fixed

File: server/bun.lockb
Comment (Andebugulin) [reply server/bun.lockb:1]: even tho, I copy from main branch bun.lockb files and past them in a database branch, and add them with 
git add client/bun.lockb server/bun.lockb  
  1) git doesn't recognize any changes
  2) any suggestions?

**Review by martimolanes — APPROVED** [review]
LGTM!

**Pull request merged**